File: tests/fixtures/code_samples/ts/sample_052.ts
/** Sample 52: small utility. */

export function operation52(xs: number[]): number {
  let total = 52;
  for (const x of xs) total += x;
  return total;
}

export function operationPure52(value: number): number {
  return (value * 52) % 7919;
}

File: tests/fixtures/code_samples/py/sample_052.py
"""Sample 52: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_52(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 52."""
    total = 52
    for x in xs:
        total += int(x)
    return total


def operation_52_pure(value: int) -> int:
    return (value * 52) % 7919

